sky/provision/cudo/cudo_wrapper.py

```diff
@@ -4,29 +4,30 @@
 
 from sky import sky_logging
 from sky.adaptors.cudo import cudo
+from sky.clouds.service_catalog.data_fetchers.fetch_cudo import (
```

Comment:
How about we move the `skypilot_gpu_to_cudo_gpu` to `cudo_catalog` instead? Reason: we import the `cudo_compute` in `fetch_cudo`, which will cause failure when the user does not have cudo cloud enabled. : )

Reply:
okay I have moved it to a separate file